views/json/page/default.php

```diff
@@ -11,10 +11,3 @@
 elgg_set_http_header("Content-Type: application/json;charset=utf-8");
 
 echo $vars['body'];
-
-// backward compatibility
-global $jsonexport;
```

Comment:
its deprecated... so it can be removed in master am i right? the other ticket can take care of the leftovers